tests/cypress/integration/skeletons_pipeline.js

```diff
@@ -166,14 +169,39 @@ context('Manipulations with skeletons', () => {
             cy.get(selector).should('not.exist');
         }
 
-        it('Creating and removing a skeleton shape', () => {
+        it.skip('Creating and removing a skeleton shape', () => {
```

Comment:
Why skipping? Looks like good basic test.
btw why we also need to `cy.removeAnnotations()` after deleting skeleton, it seems we dont save it anyway

Reply:
Oh, forgot to delete ``skip``

> why we also need to cy.removeAnnotations()

To reset object IDs, and be sure that the next created object will have client ID === 1